.pre-commit-config.yaml

```diff
@@ -9,3 +9,9 @@ repos:
     rev: v0.7.3
     hooks:
       - id: ruff
+
+-   repo: https://github.com/adamchainz/django-upgrade
```

Comment:
Move this to its own PR for discussion.

Reply:
This was a residue from https://github.com/jazzband/django-waffle/pull/528 . It's gone now.